app/model/Filters.scala

```diff
@@ -18,22 +18,23 @@ package model
 
 
 sealed trait ApiFilter[+A]
-case class Value[A](a: A) extends ApiFilter[A]
+case class Value[A](a: A, b: A) extends ApiFilter[A]
```

Comment:
Any value in making the parameters of this case class a little more descriptive?
I like A and I like B but they don't mean much out of context ;)